cfme/tests/v2v/test_v2v.py

```diff
@@ -0,0 +1,64 @@
+"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
+import fauxfactory
+import pytest
+
+from cfme.utils.log import logger
+
+pytest.mark.ignore_stream('5.8', '5.9')
+
+
+def test_basic_navigation(appliance):
+    """Test takes an appliance and tries to navigate and fill the form for Infra Map and Migration Plan.
+
+    This is a dummy test just to check navigation.
+    """
+    form_data = {
+        'general_view':
+            {
+                'name': 'infra_map_{}'.format(fauxfactory.gen_alphanumeric()),
+                'description': fauxfactory.gen_string("alphanumeric", length=50)
+            },
+        'clusters_view':
+            {
+                'mappings': [
+                    {
+                        'sources': ['Datacenter \ Cluster'],
+                        'target': ['Default \ Default']
+                    }
+                ]
+            },
+        'datastores_view':
+            {
+                'Cluster (Default)': {
+                    'mappings': [
+                        {
+                            'sources': ['NFS_Datastore_1'],
+                            'target': ['hosted_storage']
+                        },
+                        {
+                            'sources': ['h02-Local_Datastore-8GB', 'h01-Local_Datastore-8GB'],
```

Comment:
**Required**

These values cannot be hardcoded and need to be compiled form the cfme_data yaml file.